sites/statics/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@microsoft/fast-production-sites",
+  "version": "1.0.0",
+  "description": "These packages are required to run node server on Azure\"",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/fast-dna.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "private": true,
+  "dependencies": {
+    "path": "^0.12.7",
```

Comment:
You shouldn't need to enumerate "path" as a dependency. The `path` module referenced in `server.js` is a built-in module: https://nodejs.org/api/path.html